stories/topics/proc-create-credentials-cmd-generator.adoc

```diff
@@ -0,0 +1,59 @@
+[id="prod-create-credentials-cmd-generator"]
+
+= Create credentials for the Command Generator
+
+You must have an AWS credentials file for all day-2 operations, such as adding and removing Extension Nodes.
+
+The file must follow the format of the AWS credentials file for "Long-term credentials", which includes the `aws_access_key_id` and `aws_secret_access_key` variables. For further information, read link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html[AWS Command Line Interface User Guide] in the AWS documentation.
+
+An example of the credentials file format is:
```

Comment:
I would swap the line 'An example of  and the following NOTE, it looks odd saying the format is: and then having a note.

Reply:
did it